images/linux/Ubuntu1804-Readme.md

```diff
@@ -24,15 +28,15 @@
 
 ### Package Management
 - cpan 1.64
-- Helm 3.8.2
+- Helm 3.8.1
```

Comment:
need to figure out the downgrade

Reply:
https://github.com/helm/helm/issues/10948